### PR TITLE
[AN-174] Add centaur test for 4-byte unicode chars in stderr for GCP Batch

### PR DESCRIPTION
**Pull request by salonishah11**
### Description

https://broadworkbench.atlassian.net/browse/AN-174

This PR adds a centaur test for ensuring GCP Batch doesn't make a workflow fail when there are 4-byte unicode characters in stderr. Currently GCP Batch doesn't seem to include stderr content in `statusEvents` in operations metadata and hence there is no need to add special handling for these characters. But if in future this behavior is changed the centaur test should warn us of it when it starts failing.

#### `CHANGELOG.md`
 - [ ] I updated `CHANGELOG.md` in this PR
 - [x] I assert that this change shouldn't be included in `CHANGELOG.md` because it doesn't impact community users

#### Terra Release Notes
 - [ ] I added a suggested release notes entry in this Jira ticket
 - [x] I assert that this change doesn't need Jira release notes because it doesn't impact Terra users

### CODE REVIEW

File: centaur/src/main/resources/standardTestCases/emoji_in_stderr/emoji_in_stderr.wdl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version 1.0
+
+task makeAnEmoji {
+    command {
+        AN_EMOJI='\360\237\246\204\n'
```
Comment (LizBaldo) [inline centaur/src/main/resources/standardTestCases/emoji_in_stderr/emoji_in_stderr.wdl:5]: Unicorn emoji 🎉

**Review by LizBaldo — APPROVED** [review]
LGTM, glad this is not an issue currently, and that the test should warn us when it becomes an issue :) :shipit:

**Comment by salonishah11** [comment]
Turns out the centaur test won't fail in the expected way if GCP Batch starts including stderr content in `statusEvents`. Hence closing this PR.
